lib/from_openstudio/model.rb

```diff
@@ -178,5 +201,49 @@ def self.materials_from_model(openstudio_model)
       result
     end
 
+    # Create HB Construction from OpenStudio Materials
+    def self.constructions_from_model(openstudio_model)
+      result = []
+
+      # Create HB AirConstruction from OpenStudio Construction
+      openstudio_model.getConstructionAirBoundarys.each do |construction|
+        result << AirBoundaryConstructionAbridged.from_construction(construction)
+      end
+
+      # Create HB WindowConstruction from OpenStudio Construction
+      openstudio_model.getConstructions.each do |construction|
+        if construction.isFenestration
```

Comment:
Nice. That's a useful property of the OpenStudio SDK. However, I guess that you still have to check for the case that it's an opaque door construction (that still counts as fenestration). Maybe we should just forego this check and only stick to check the nature of the material layers.